Examples/testCustomEntitlementsComputation/README.md

```diff
@@ -0,0 +1,32 @@
+# Test Custom Entitlements Computation App
```

Comment:
Hmm the `test` part on the name of the app could be confusing, since this is not running tests or anything... How about `Custom entitlements computation sample app`?